pkg/loop/internal/pb/ocr3/contract_transmitter.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb/ocr3;ocr3pb";
+
+package loop.internal.pb.ocr3;
+
+import "google/protobuf/empty.proto";
+import "ocr3_reporting.proto";
+
+message Signature {
+  bytes signature = 1;
+  uint32 signer = 2; // NOTE: this is actually a uint8, but proto doesn't support this.
+}
+
+message TransmitRequest {
+  bytes configDigest = 1;
```

Comment:
is this a fixed len? if so comment please (iiuc the impl is erroring if not len 32 )